CHANGES.md

```diff
@@ -27,6 +27,25 @@ OpenSSL 3.2
 
 ### Changes between 3.2.0 and 3.2.1 [xx XXX xxxx]
 
+ * A file in PKCS12 format can contain certificates and keys and may come from
+   an untrusted source. The PKCS12 specification allows certain fields to be
+   NULL, but OpenSSL does not correctly check for this case. This can lead to a
+   NULL pointer dereference that results in OpenSSL crashing. If an application
+   processes PKCS12 files from an untrusted source using the OpenSSL APIs then
+   that application will be vulnerable to this issue.
```

Comment:
It's a bit strange that the formulation still makes it seem like 3.2.1 is vulnerable. I'm guessing this is copied straight from the pre-3.2.1 CVE [announcement](https://mta.openssl.org/pipermail/openssl-announce/2024-January/000292.html) text? I think it would be better to write "Before 3.2.1, this could lead to a NULL pointer dereference ..."